server/api/api.go

```diff
@@ -96,7 +94,6 @@ func (a *API) RegisterRoutes(r *mux.Router) {
 	a.registerBoardsRoutes(apiv2)
 	a.registerBlocksRoutes(apiv2)
 	a.registerContentBlocksRoutes(apiv2)
-	a.registerStatisticsRoutes(apiv2)
```

Comment:
Who was using this statistics? Are we certain it's no longer needed?

Reply:
Nice catch, we need this. Reverting the changes back.